.github/workflows/pre-commit.yml

```diff
@@ -38,33 +73,64 @@ jobs:
       run: |
         python -m pytest src/test/python
 
-  lint:
-    name: Lint ${{ matrix.package }}
+  lintpython27:
+    name: Lint ${{ matrix.package }} - Python27
 
     runs-on: ubuntu-latest
     strategy:
       max-parallel: 4
       matrix:
-        package: [tools]
-        
+        package: [common, libs, platform]
+
     steps:
       - name: Checkout ${{ matrix.package }}
         uses: actions/checkout@v1
-    
+
       - name: Set up Python 2.7
         uses: actions/setup-python@v1
         with:
           python-version: 2.7
-   
+
       - name: Install flake8
         run: |
           python -m pip install --upgrade pip
           pip install flake8
-        
+
       - name: Run flake8 on src directory
         working-directory: ${{ matrix.package }}
         run: python -m flake8 src/main/python --max-line-length 88
-    
+
+      - name: Run flake8 on test directory
+        working-directory: ${{ matrix.package }}
+        run: python -m flake8 src/test/python --max-line-length 88
+
+  lintpython38:
+    name: Lint ${{ matrix.package }} - Python38
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        package: [tools, common, libs, platform]
```

Comment:
super nit: can we do these 4 in the same order everywhere (ideally alphabetical)

Reply:
Sure will do